Components/SystemDefines.hpp

```diff
@@ -98,7 +98,7 @@ constexpr uint8_t BATTERY_TASK_RTOS_PRIORITY = 2;            // Priority of the
 constexpr uint8_t BATTERY_TASK_QUEUE_DEPTH_OBJS = 10;        // Size of the battery voltage task queue
 constexpr uint16_t BATTERY_TASK_STACK_DEPTH_WORDS = 512;        // Size of the battery voltage task stack
 
-constexpr uint32_t TELEMETRY_DEFAULT_LOGGING_RATE_MS = 500; // Default logging delay for telemetry task
+constexpr uint32_t TELEMETRY_DEFAULT_LOGGING_RATE_MS = 333; // Default logging delay for telemetry task
```

Comment:
only make this change after it has been tested. do not integrate this change into this pull request.

Reply:
This code was tested during tabletop